### ISSUE TITLE
New logo and branding

### ISSUE THREAD

**Issue by joaobborges** [JosefNemec/Playnite#1693]
**Describe the solution you'd like**
Design a beautiful logo that doesn't feel so generic :)

**Comment by comradekingu** [comment]
What about an old style joystick in the form of something that looks like a lamppost with round light source?

**Comment by ArtexJay** [comment]
![image](https://user-images.githubusercontent.com/32932497/91510884-52d60c80-e8ac-11ea-983b-736a2b31e602.png)

I made this for myself, I have two designs I prefer this one. I tried to do something close to the new fluent icons but ended up with this lol.

**Comment by ArtexJay** [comment]
I also made this monstrosity. I liked it at first but not crazy about it tbh.
![applogo](https://user-images.githubusercontent.com/32932497/91511048-bc561b00-e8ac-11ea-8946-3e536995f3bc.png)

**Comment by comradekingu** [comment]
![image](https://user-images.githubusercontent.com/13802408/91555351-287a6280-e931-11ea-8870-ba8730ff74ad.png)

To illustrate an idea. Was thinking more head on, or upwards, but didn't see any such images.

**Comment by Bluscream** [comment]
Wtf is that thing? Looks like a abomination of Sirenhead

**Comment by shangles** [comment]
Agreed.  None of these are better than the original logo.  They aren't made using vector-based illustration software and won't translate to print or any other media well.

**Comment by comradekingu** [comment]
@shanges @Bluescream The idea is to have the top of the joystick look like a lit lamppost.
Thereby the "night" is incorporated into the "play" of Playnite. From that it is easier to simplify rather than arrive at some idea by refining a generic concept.

**Comment by JosefNemec** [comment]
`nite` in Playnite is for `unite`. Playnite name is a combination of `play` and `unite`.

**Comment by darklinkpower** [comment]
I think this is better left to someone who does graphic design. The only issue isn't the making of the logo itself in a software designed for it, but the difficult part is coming with a good concept and creative process to make a good logo. I agree that the current logo feels generic but not just because of that it should be replaced for anything that doesn't have quality.

**Comment by comradekingu** [comment]
@JosefNemec A play-night, like a play-date or game-night isn't a solitary event either.
@darklinkpower Contrary to the mockup on display, I actually do some graphic design. ;)
https://www.flickr.com/photos/neilg24/4018550872 is a very iconic lamp-post, and it has a rounder shape.
I suppose the width of the pillar lends itself more to arcade style joysticks.

**Comment by darklinkpower** [comment]
Personally I think your concept is far from a good idea for a logo for Playnite. To name a few reasons:
1.  You are talking about joysticks when Playnite doesn't focus exactly on retro games and emulator support is still missing important features as seen in #653. Defining the Playnite focusing in one aspect of gaming is a terrible idea in my opinion, specially as @JosefNemec has said that the concept behing "Playnite" is "unite". The concepts used for the name shouldn't be ignored. A joystick-like icon _could probably_ go with other projects like RetroPie, Launchbox or RetroArch for example.
2. A joystick feels extremely generic and doesn't have a personality or can be instantly recognizable to one thing, which would fall in the same "generic" category as the controller, which would make the change pointless
3. Since it's been just ideas, I don't know if this would apply but it would be important to see how would be the type of logo: Minimalistic, logo+typography, lettermarks, emblems, etc. And then there's the visual style of the logo, which is a huge subject.

If you have education in design or art, I'm sure you'll be able to come with other ideas easily but I think it would be a good idea to put here semi-finished logos because in abstract it is difficult to value them. But that's just my input as a Playnite user, the final word falls of course in @JosefNemec as the Playnite developer.

**Comment by shakeyourbunny** [comment]
I'd like to add, that the main platform for games is the PC. On the PC, the main gaming input device never has been gamepads nor joysticks (except flight or space simulators), but mouse and keyboard. The current logo design is imho ok, because it symbolizes a XBOX gamepad which most people have or something similar which has XInput.

I understand that there is a certain userbase that use Playnite primarily as a unifying frontend for console emulation (and to a lesser degree, computer emulation), but as Darklin mentionend (and the very first thing you see on https://playnite.link) that Playnite is not focused on that, but a neutral unifying platform for all of your games. This also PC gaming in its various forms (including traditional gaming without any digital libraries).

I think that the current logo design is good enough for that, but the ultimate decision to update the branding is on @JosefNemec .

**Comment by shakeyourbunny** [comment]
Oh and your proposed joystick design just looks like the Commodore 64 Direct-to-TV (C64 DTV). See https://en.wikipedia.org/wiki/C64_Direct-to-TV

**Comment by comradekingu** [comment]
It is an idea, not a proposal. I just searched for AMIGA controller and overlaid it on some other image.
Trying to think of something visual around "unite".

**Comment by ArtexJay** [comment]
> Agreed. None of these are better than the original logo. They aren't made using vector-based illustration software and won't translate to print or any other media well.

I can do it as a vector based just fine, but it's faster for me to whip up something in photoshop quickly especially if it's concept. of course I'm biased but I rather like my the idea I posted. But the whole unite thing is giving me an idea.

**Comment by ArtexJay** [comment]
> Personally I think your concept is far from a good idea for a logo for Playnite. To name a few reasons:
> 
> 1. You are talking about joysticks when Playnite doesn't focus exactly on retro games and emulator support is still missing important features as seen in #653. Defining the Playnite focusing in one aspect of gaming is a terrible idea in my opinion, specially as @JosefNemec has said that the concept behing "Playnite" is "unite". The concepts used for the name shouldn't be ignored. A joystick-like icon _could probably_ go with other projects like RetroPie, Launchbox or RetroArch for example.
> 2. A joystick feels extremely generic and doesn't have a personality or can be instantly recognizable to one thing, which would fall in the same "generic" category as the controller, which would make the change pointless
> 3. Since it's been just ideas, I don't know if this would apply but it would be important to see how would be the type of logo: Minimalistic, logo+typography, lettermarks, emblems, etc. And then there's the visual style of the logo, which is a huge subject.
> 
> If you have education in design or art, I'm sure you'll be able to come with other ideas easily but I think it would be a good idea to put here semi-finished logos because in abstract it is difficult to value them. But that's just my input as a Playnite user, the final word falls of course in @JosefNemec as the Playnite developer.

This is something i had in mind, two hands over each other forming a gamepad something like this. Did this one quickly just so I could see and show more or less what I am thinking of.
![image](https://user-images.githubusercontent.com/32932497/92547809-bf91c500-f223-11ea-8fb8-d1e3e35c5bfa.png)

**Comment by shakeyourbunny** [comment]
This looks like a symbol for a Nintendo thing, not a PC.

**Comment by shakeyourbunny** [comment]
I put together a quick drawing which IMHO suits Playnite much better for the motif of "unity" "gaming" and PC.

![playnite-unity-logo1](https://user-images.githubusercontent.com/5352403/93663535-04f69380-fa69-11ea-89b5-62e4e70d6aae.png)

Alternative:

![playnite-unity-logo2](https://user-images.githubusercontent.com/5352403/93663541-0d4ece80-fa69-11ea-8ff7-342f47e2c2dc.png)

Although I think that the current logo is fine though, but may be a bit better if it has a slight drop shadow and the edges are closed with a tiny black border.

**Comment by shakeyourbunny** [comment]
I'd really like to add more out of the box supported libraries, but this will be cluttered with undecipherable pixels and the Battle.net logo shoulda be black/white too, but could not edit the logo properly.

If someone has interest building on this, I'd happily provide the .svg version of both.

**Comment by shakeyourbunny** [comment]
I'm not an graphics artist, but to sell the notion of "unity", some ideas:
- the logo should not be overcomplicated to have brand recognition
- perhaps just the current Playnite logo (with a 1 pixel black border) in the center and some circle or rounded square.
- on the rounded square or circle there are some (colored?) balls / rounded squares representing the libraries
...
or just at the bottom a row of (slightly) (coloured) squares for the libraries, encircled with a rounded rectangles, where atop is the current playnite logo.

**Comment by darklinkpower** [comment]
A logo definitely shouldn't be so busy and definitely shouldn't use other logos as part of its logo. This task should be better left for a graphic designer. Graphic design is more difficult than what it appears (And it's a profession on its own), there are a lot of things involved in the creative process and it's not just a matter of making the product in a software. There's no rush in changing the logo, the current one woks fine enough and the change (in my opinion) should only be made when there's a professional quality logo done by a graphic designer. My career is design oriented and I could try giving it a go, but even then I doubt a lot that my proposals would have the quality of a something done by a professional graphic designer.

Maybe there could be a small announcement in Playnite 8 release about this to potentially grab interest and see if someone can contribute.

**Comment by shakeyourbunny** [comment]
@darklinkpower 
I wholeheartly agree to your answer, my proposition has more the reason to get people off the emulator joypad train and nudge them more to the vision @JosefNemec  has, as I clearly am not an graphics artist ;)

**Comment by JosefNemec** [comment]
Here are my thoughts:
- The logo can't be very detailed because it has to look good when shrunken down to icon size.
- The logo doesn't have to have "play", "unite" or any other relevant theme. There are plenty of apps that have abstract logos that are no direct references to what the app is about.
- I'm not a fan of any proposed variants.

I don't think it's worth making just a new logo. Playnite could use complete branding overhaul, including new website, possibly also refresh of default application theme. Also, ideally tied to some major update release.

**Comment by Bluscream** [comment]
![image](https://user-images.githubusercontent.com/3318223/94187810-cae22300-fea8-11ea-8940-7033387a0635.png)

**Comment by felixkmh** [comment]
I experimented with some logo/icon ideas while working on a theme. The idea was to somehow incorporate the "play triangle": 
![grafik](https://user-images.githubusercontent.com/24227002/116367261-740a4680-a807-11eb-94a2-6ef3d6bacf10.png)
